ec3730/Models/DataUsageModel.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  DataUsageModel.swift
+//  ec3730
+//
+//  Created by Ahmad Azam on 21/08/2022.
+//  Copyright © 2022 Zachary Gorak. All rights reserved.
+//
+
+import Foundation
+
+extension SystemDataUsage {
+    static var wifiCompelete: String {
```

Comment:
Misspelled.  Usage `wifiTotal` instead. Same for wwan.

Reply:
Right, will update